CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Contributing guidelines for the project
+
+First off, thank you for considering contributing to this project. It's people like you that make this project such a great tool. We appreciate your help!
+
+The following is a set of guidelines for contributing to this project. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Introduction
+
+This project is using TypeScript and [OCLIF](https://oclif.io/) to build the CLI. The project is using Jest for testing, ESLint for code formatting and linting.
+
+## Issues and feature requests
+
+If you have any issues or feature requests, please create an issue in the [GitHub issues](https://github.com/nimblehq/infrastructure-templates/issues) page.
+
+## Development
+
+### Prerequisites
+
+- Node.js 12.0.0 or higher
+- NPM is preferred to use with this project, but you can use Yarn as well and don't commit the `yarn.lock` file to the repository
+
+### Install dependencies
+
+```bash
+npm install
+```
+
+### Run the CLI locally
+
+Run the following command and follow the instructions:
+
+```bash
+bin/dev generate {project-name}
+```
+
+### Project structure
+
+The project has the following main folders:
+
+- `bin` - contains the executable file for the CLI
+  - `dev` - the executable file for the CLI in development mode
+  - `run` - the executable file for the CLI in production mode
+
+- `src` - contains the source code of the CLI
+  - `index.ts` - the entry point of the CLI
+  - `commands` - contains the commands of the CLI
+  - `helpers` - contains the helper functions of the CLI
+    - `childProcess` - contains the helper functions for child process
+    - `file` - contains the helper functions for file interaction
+    - `terraform` - contains the helper functions for Terraform
+  - `templates` - contains the code to generate the templates
+    - `addons` - contains the code to generate the common addons e.g. version control, CI/CD, etc.
+    - `aws` - contains the code to generate the AWS modules e.g. VPC, RDS, etc.
+    - `core` - contains the code to generate the main Terraform files e.g. `main.tf`, `variables.tf`, etc.
+  - `skeleton` - contains the skeleton files for the project
+    - `addons` - contains the skeleton files for common addons e.g. version control, CI/CD, etc.
+    - `aws` - contains the skeleton files for AWS modules
+    - `core` - contains the skeleton files for common files e.g. `main.tf`, `variables.tf`, `.gitignore`, etc.
+  - `test` - contains the test helpers and configurations
+
+### Add a new command
+
+To add a new command, you need to create a new folder in the `src/commands` folder and add the `index.ts` file in it.
```

Comment:
maybe we should say something like this
```suggestion
To add a new command, create a new folder in the `src/commands` folder and add the `index.ts` file.
```

Reply:
Modified in https://github.com/nimblehq/infrastructure-templates/pull/141/commits/1b6304955c0ca8410a0a48d67a6894b2b425383e